c2rust-analyze/tests/filecheck/fields.rs

```diff
@@ -52,8 +52,10 @@ struct VecTup<'a> {
 // CHECK-DAG: assign Label { origin: Some(Origin([[P_REF_A_ORIGIN]]))
 
 // CHECK-LABEL: final labeling for "_field_access"
-// CHECK-DAG: ([[@LINE+3]]: ppd): addr_of = UNIQUE, type = READ | WRITE | UNIQUE
-// CHECK-DAG: ([[@LINE+2]]: ra): &mut A
+// CHECK-DAG: ([[@LINE+5]]: ppd): addr_of = UNIQUE, type = READ | WRITE | UNIQUE
```

Comment:
just to be clear, it's rewrites of the fields that are currently not supported by this PR, but the ADT type itself should be covered?

Reply:
Currently neither field expressions (e.g. `my_struct.x`) nor field type declarations (e.g. `x: *mut i32`) get rewritten.